### DISCUSSION TITLE
Some questions about QNH

### DISCUSSION THREAD

**Discussion by Stefaandk** [flybywiresim/aircraft#2370] [Help]
Hi,

Flying in Europe atm, I get the metar for Gatwick/EGKK which came back with 985hpa.

So I can put the glarescreen to either inHg or hPa but I'm confused about entering the metar value in the MCDU approach.
Is the one in there in inHg or hPa (does it matter, does the plane know what I'm using?)  So yesterday I enter 985 in the MCDU approach data.  Then below 10K feet I change the glareshield unit to hPa and also 985 as per metar (using in real weather in sim).  When on final though my altitude showing on the PFD actually went below zero.  Bit confused what's going on here and what I should set in the MCDU and glareshield given the metar data provided.

Edit: Furthermore given this weird behaviour I hit B on the keyboard and it put the pHa back to 1013, so was the sim ignoring its own weather/metar data?

Thx

**Comment by Figrol** [comment]
Where are you getting the weather from? Through the Sim or online? If it's online, likely meant the SIM had the wrong meats, since 1013 is the standard QNH it probably that the SIM hadn't picked it up. From what I understand, the aircraft can take the altitude date in hPa or inHg, remember seeing it in one of the pull request. (Possibly depending which version of FBW you are using)
